kafka-elastic/src/test/java/com/baeldung/kafka/service/ElasticSearchServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.kafka.service;
+
+import com.baeldung.kafka.entity.NotificationEntity;
+
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+
+import static org.mockito.Mockito.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.MockitoAnnotations;
+
+class ElasticSearchServiceTest {
```

Comment:
Test names should be one of:
https://github.com/eugenp/tutorials/blob/efc934d2601fff7f0766c59c8766593fcce359f7/custom-pmd/src/main/java/com/baeldung/pmd/UnitTestNamingConventionRule.java#L18

(same elsewhere)

Tests that use Mockito would be UnitTest

Reply:
incoprated